app/routes.js

```diff
@@ -60,6 +60,16 @@ router.get('/solutions/:filterType', async (req, res, next) => {
   }
 });
 
+router.post('/solutions/custom', async (req, res, next) => {
+  const capabilitiesSelected = req.body.capabilities;
```

Comment:
can this be changed to
`const { capabilities: capabilitiesSelected } = req.body;`?

Reply:
this is really splitting hairs :D I prefer it like it is tbh